.NET/Microsoft.Recognizers.Definitions.Common/Microsoft.Recognizers.Definitions.Common.csproj

```diff
@@ -412,6 +412,10 @@
     <DotNetCliToolReference Include="T5.TextTransform.Tool" Version="1.1.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I had removed this before. Please don't re-add spurious changes to project files.

Reply:
@LionbridgeCS Please fix this.